### DISCUSSION TITLE
cannot remove entry

### DISCUSSION THREAD

**Discussion by LucasZhan** [facebookresearch/hydra#2152] [Q&A] [answered]
## Description

I am new to hydra and familiar with it via tutorials. My current pwd is:
**hydra/examples/tutorials/basic/your_first_hydra_app/5_defaults**

My issue is:
Cannot remove entry by command **python my_app.py ~db**

all the code is cloned from github and have not been modified.

## Checklist
- [x] I checked on the latest version of Hydra
- [x] I created a minimal repro (See [this](https://stackoverflow.com/help/minimal-reproducible-example) for tips).

## To reproduce

** Minimal Code/Config snippet to reproduce **

1. clone this repo.
2. cd **hydra/examples/tutorials/basic/your_first_hydra_app/5_defaults**
3. run **python my_app.py ~db**

** Stack trace/error message **
```
zsh: no such user or named directory: db
```

## Expected Behavior

It should remove a entry.

## System information
- **Hydra Version** :  1.1.2
- **Python version** : 3.8.12
- **Virtual environment type and version** :  conda & conda 4.10.3
- **Operating system** :  Mac OS 12.3.1

**Comment by LucasZhan** [comment]
Note for your convenience: I am using M1 chips.

**Comment by Jasha10** [comment, marked as answer]
Hi @LucasZhan,
I think your shell (zsh) is interpreting the tilde `~` in a special way.
Please try quoting the `~db` value as `'~db'`:
```
python my_app.py '~db'
```
See the Hydra docs on quoting: https://hydra.cc/docs/advanced/override_grammar/basic/#working-with-your-shell

**Reply by LucasZhan** [reply]
Thanks for your quick reply and happy Easter day!
